### DISCUSSION TITLE
Move the osu! AppData/Local folder to the Linux version

### DISCUSSION THREAD

**Discussion by SIUUUUUUUUUU17** [ppy/osu#30777] [Q&A] [answered]
Is this possible? I'm looking to import my stuff over from my Windows install

**Comment by Stoppedpuma** [comment, marked as answer]
You can do this by copying the `osulazer` folder onto your Linux install and specifying the location with the "Change folder location" option. If you plan on using the default location, replace the existing folder at `~/.local/share/osu`.

![image](https://github.com/user-attachments/assets/2ca89eb9-4dd3-4cd5-bb5d-9c7533b60754)
